### PR TITLE
Updated B2C sample to latest MSAL and updated readme.md

### PR DESCRIPTION
**Pull request by oldalton**

### CODE REVIEW

File: MSALiOSB2C/ViewController.swift [hunk 1]
```diff
@@ -66,7 +66,9 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
              The scheme part, i.e. "msal<your-client-id-here>", needs to be registered in the info.plist of the project
              */
             
-            let pcaConfig = MSALPublicClientApplicationConfig(clientId: kClientID)
+            let authority = try self.getAuthority(forPolicy: self.kSignupOrSigninPolicy)
+            
+            let pcaConfig = MSALPublicClientApplicationConfig(clientId: kClientID, redirectUri: nil, authority: authority)
```
Comment (unpluggedk) [inline MSALiOSB2C/ViewController.swift]: nit: do we want to specify redirectUri or put in a comment what nil means here?

File: MSALiOSB2C/ViewController.swift [hunk 2]
```diff
@@ -104,7 +106,8 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
              flow completes, or encounters an error.
              */
             
-            let parameters = MSALInteractiveTokenParameters(scopes: kScopes)
+            let webViewParameters = MSALWebviewParameters(parentViewController: self)
+            let parameters = MSALInteractiveTokenParameters(scopes: kScopes, webviewParameters: webViewParameters)
             parameters.authority = authority
             application.acquireToken(with: parameters) { (result, error) in
                 if let result = result {
```
Comment (unpluggedk) [inline MSALiOSB2C/ViewController.swift]: nit: I think guard let and do error handling first might look better, but no harm in not changing :)

Comment (oldalton) [reply MSALiOSB2C/ViewController.swift]: addressed

File: MSALiOSB2C/ViewController.swift (hunk 1 above)
Comment (oldalton) [reply MSALiOSB2C/ViewController.swift]: addressed

**Pull request merged**